Services/Database/classes/PDO/class.ilDBPdoPostgreSQL.php

```diff
@@ -25,6 +25,7 @@ public function generateDSN() : void
                      . $this->getUsername() . ';password=' . $this->getPassword();
     }
 
+
```

Comment:
@iszmais Did the `PHP-CS-FIXER` remove these empty lines? If not please, revert the change (here and everywhere else).l